_posts/2018-11-01-managing-translations-with-crowdin.md

```diff
@@ -0,0 +1,148 @@
+
+# Table of contents
+1. [What is Crowdin](#what-is-crowdin)
+2. [Key features](#key-features)
+3. [Getting started](#getting-started)
+4. [In app translations](#in-app-translations)
+5. [Conclusion](#conclusion)
+
+## What is Crowdin
+Crowdin is an online platform that allows you to do translations through a visual interface.
+You can appoint people as translators for any natural language.
+Either an integration with your Git repository or a CLI tool can be used to get the source strings onto Crowdin.
+After uploading the source strings, the translators can get to work.
+They'll be notified whenever there's something new to translate.
+When they're done, approvers can start proof-reading with the help of some thorough quality assurance functionalities.
+You'll never miss a translation again because it will clearly be marked as not being translated.
+Strings that are removed from the source file will also be removed from all translations.
+The translated files can be downloaded at any moment and will be in the exact same format as the source file.
+In other words, clean and up to date translation files!
+
+Translator view:
+<img class="image fit" style="margin:0px auto; max-width: 1422px;" alt="Crowdin translator view" src="/img/2018-11-02-managing-translations-with-crowdin/translator_mode.png">
+
+Approver view:
+<img class="image fit" style="margin:0px auto; max-width: 1439px;" alt="Crowdin approver view" src="/img/2018-11-02-managing-translations-with-crowdin/approver_mode.png">
+
+## Key features
+* A large number of file formats are supported, ranging from `json` files to `csv` and `properties` files.
+It can thus be used for Angular, React, Vue ... applications as well as a Spring Boot service and so on.
+* Crowdin has a very clever parser and detects thing such as HTML tags and variables.
+It will warn you if they are translated and they no longer match the original tags/names.
+* Quality assurance checks will help the approver in verifying the translations.
+* Extra context can be provided for each source string to help translators understand what exactly they need to translate.
+* Integration with source control systems is very easy and will make sure that the source strings are up to date with your Git repository. 
+Translations will automatically be pushed to your Git repository.
+* There's even the possibility to integrate it with your website and have people do the translations on your website directly.
+That way the translators get a lot of context of what to translate.
+* If you're unable to do the translation for a certain language, 
+Crowdin allows you to easily hire professional translators!
+
+## Getting started
+### Account
+To start using Crowdin, you of course need an account at [Crowdin.com](https://crowdin.com). 
+There's a limited free trial. 
+After that you can choose from a variety of payment plans depending on the number of source strings and projects you'll have.
```

Comment:
What does a project signify? 1 project = 1 Git repo? So an application's backend and frontend each having their own Git repo would make it two projects? Perhaps you could clarify that here.